src/components/VCopyButton.vue

```diff
@@ -16,49 +16,64 @@
   </VButton>
 </template>
 
-<script>
+<script lang="ts">
 import Clipboard from 'clipboard'
 
+import {
+  defineComponent,
+  onBeforeUnmount,
+  onMounted,
+  ref,
+} from '@nuxtjs/composition-api'
+
 import VButton from '~/components/VButton.vue'
 
-export default {
+import type ClipboardJS from 'clipboard'
```

Comment:
Thank you! I tried joining both imports, but since they both appear to be default exports, decided to leave as is. Didn't realize that `Clipboard` could be used as type, too.